cmd/run.go

```diff
@@ -35,6 +35,7 @@ var (
 	}
 )
 
+//nolint
```

Comment:
Why no lint?

Reply:
Linter has rule for init functions `gochecknoinits` which is annoying. Probably should figure out something better.